src/cpp/QtWidgets/QDial/qdial_wrap.cpp

```diff
@@ -0,0 +1,96 @@
+
+#include "qdial_wrap.h"
+#include "src/cpp/QtWidgets/QWidget/qwidget_wrap.h"
+#include "src/cpp/Extras/Utils/nutils.h"
+#include <QWidget>
+
+
+Napi::FunctionReference QDialWrap::constructor;
+
+Napi::Object QDialWrap::init(Napi::Env env, Napi::Object exports) {
+  Napi::HandleScope scope(env);
+  char CLASSNAME[] = "QDial";
+  Napi::Function func = DefineClass(env, CLASSNAME, {
+    InstanceMethod("setNotchesVisible", &QDialWrap::setNotchesVisible),
+    InstanceMethod("setWrapping", &QDialWrap::setWrapping),
+    InstanceMethod("setNotchTarget", &QDialWrap::setNotchTarget),
+    InstanceMethod("notchTarget", &QDialWrap::notchTarget),
+    InstanceMethod("notchesVisible", &QDialWrap::notchesVisible),
+    InstanceMethod("wrapping", &QDialWrap::wrapping),
+    QWIDGET_WRAPPED_METHODS_EXPORT_DEFINE(QDialWrap)
```

Comment:
QWIDGET_WRAPPED_METHODS_EXPORT_DEFINE is both here and inside the QABSTRACTSLIDER_WRAPPED_METHODS_EXPORT_DEFINE macro. 

Maybe its a good idea to remove it from here.

Reply:
Ohhh yeah, you are totally right. Done :)